contracts/token/ERC20/extensions/ERC4626.sol

```diff
@@ -17,28 +17,45 @@ import "../../../utils/math/Math.sol";
  * the ERC20 standard. Any additional extensions included along it would affect the "shares" token represented by this
  * contract and not the "assets" token which is an independent contract.
  *
- * CAUTION: When the vault is empty or nearly empty, deposits are at high risk of being stolen through frontrunning with
- * a "donation" to the vault that inflates the price of a share. This is variously known as a donation or inflation
+ * CAUTION: In empty (or nearly empty) ERC-4626 vaults, deposits are at high risk of being stolen through frontrunning
```

Comment:
The current way it's written, this paragraph will show up in a "Caution" box, while the next two paragraphs will show up outside of it. We can remove "CAUTION:" to remove the box completely. Alternative, we put multiple paragaphs in the box by changing the syntax to:

```
[CAUTION]
====
content here
====
```

```suggestion
 * In empty (or nearly empty) ERC-4626 vaults, deposits are at high risk of being stolen through frontrunning
```

Reply:
Made a big caution box.